portfolio/src/main/webapp/index.html

```diff
@@ -52,7 +52,14 @@ <h3>Past entries</h3>
         <option value=15>15</option>
       </select>
       <br>
-      <div id="guest-book-comments"></div> 
+      <div id="guest-book-comments"></div>
+      <h3> Delete all guestbook comments </h3>
+      <form onsubmit="deleteData();return false"> <!-- Don't reload page after -->
+        <textarea name="password" id="password" maxlength=30 placeholder="Password"></textarea>
```

Comment:
Instead of textarea you could use an input with type password so it would be more recognizable as a typical password input.

Reply:
I tried switching to an input type=password, but it seemed to break the form - it wouldn't execute the onsubmit javascript at all. Do you know why that might be?

Reply:
Hmm maybe try having deleteData return false instead of writing return false after? I tested that [here](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onsubmit) and it seemed to work but maybe there's some other bug, not sure

Reply:
I just finished a bunch of debugging and the problem turned out to be somewhere totally different than where I expected :/ Thank you, though!